docs/user_guide/fonts.rst

```diff
@@ -0,0 +1,54 @@
+Fonts and FontAwesome
+=====================
+
+The theme includes the `FontAwesome 5 Free <https://fontawesome.com/icons?m=free>`__
+icon font (the ``.fa, .far, .fas`` styles, which are used for
+:ref:`icon links <icon-links>` and admonitions).
+This is the only `vendored` font, and otherwise the theme by default relies on
```

Comment:
```suggestion
This is the only *vendored* font, and otherwise the theme by default relies on
```

this is a vacuous change, and is done for the benefit of source-code readers (who might see single backticks and think "code" or "there ought to be 2 backticks")